index.html

```diff
@@ -128,6 +128,20 @@ <h4>Suzaan clarke</h4>
 </section>
 <!-- End Our Experts Section -->
 
+<!-- Start Contact us Section -->
+<section id="contact-us">
+  <div class="container">
+      <div class="title"><h2>Contact Us</h2></div>
+      <form>
+          <input type="text" class="field" placeholder="Your Name">
+          <input type="text" class="field" placeholder="Your Email">
+          <textarea placeholder="Your Message" class="field"></textarea>
+          <button class="btn">Submit</button>
+      </form>
```

Comment:
add some functionality for these elements
suggestions: 
- after submitting a message, create a pop-up message to thank the user for his feedback
- if the user leaves a message field empty give him a message or red flag to confirm the form contain a message